ide/schema2beans/src/org/netbeans/modules/schema2beansdev/gen/JavaUtil.java

```diff
@@ -852,7 +852,7 @@ public static int getOptimialHashMapSize(Object[] keys, int maxSize) {
         int defaultAnswer = keyLength + 1;
         try {
             java.lang.reflect.Field tableField = HashMap.class.getDeclaredField("table");
```

Comment:
What about removing the reflection all together and just assuming that `keys.length * 3 / 2` is usually good enough?

Reply:
i actually wanted to deprecate the method - i forgot, Lets see if it builds, if it does I push again.

Reply:
deprecated it + added `keylength * 3 / 2` as default return value. Its what the JDK uses anyway if i remember correctly.